docs/3.0-migration-guide.md

```diff
@@ -137,17 +137,13 @@ and relied on that version in order to communicate with peers that are not up
 to date. If one of your peers is in that case, please try contacting them and
 encouraging them to upgrade their software.
 
-Remove support for compatibility with old Mbed TLS's truncated HMAC
--------------------------------------------------------------------
-
-This doesn't affect people using the default configuration as it was already
-disabled by default.
+Remove support for truncated HMAC
+---------------------------------
 
-This only affects TLS users who enabled `MBEDTLS_SSL_TRUNCATED_HMAC_COMPAT` and
-used the Truncated HMAC extension to communicate with peers using old version
-of Mbed TLS. Please consider using a CCM-8 ciphersuite instead of the
-Truncated HMAC extension, or convincing your peer to upgrade their version of
-Mbed TLS.
+This only affects TLS users who enabled `MBEDTLS_SSL_TRUNCATED_HMAC`
```

Comment:
Sorry, on second read this paragraph has a few issues:

1.  users who enabled `MBEDTLS_SSL_TRUNCATED_HMAC` -> it was enabled in the default configuration, so people didn't need to enable it.
2. syntax : missing word between the two options
3. convincing your peer to upgrade their version of Mbed TLS -> that only applied to the compat option.

Suggested wording:

This affects users of truncated HMAC, that is, users who called `mbedtls_ssl_conf_truncated_hmac( ..., MBEDTLS_SSL_TRUNC_HMAC_ENABLED)`, regardless of whether the standard version was used or compatibility version (`MBEDTLS_SSL_TRUNCATED_HMAC_COMPAT`).

The recommended migration path for people who want minimal overhead is to use a CCM-8 ciphersuite.

Reply:
Thanks @mpg ! I have used your suggested working and pushed again.